bigearthnet_encoder/encoder.py

```diff
@@ -59,23 +59,23 @@ def tiff_dir_to_ben_s1_patch(patch_dir: DirectoryPath, **kwargs):
 
 @validate_arguments
 def tiff_dirs_to_ben_s1_s2_patch(
-    s1_patch_dir: DirectoryPath, s2_patch_dir: DirectoryPath, **kwargs
+    ben_s1_path: DirectoryPath,
+    s2_patch_dir: DirectoryPath, **kwargs
 ):
+    s1_patch_name = s2_to_s1_patch_name(s2_patch_dir.name)
+    s1_patch_dir = ben_s1_path.joinpath(s1_patch_name)
```

Comment:
To keep it consistent with other parts of the code, I would rather use
```
ben_s1_path / s1_path_name
```

Reply:
I actually changed it to ben_s1_path to keep it consistent with the other parts of the code, due to the fact that we are only dealing with the general s1 directory here. This directory path is called ben_s1_path in the later functions calls. However, if you think a name change would better convey its function, feel free to change it.

Reply:
Ah ok. GitHub didn't highlighted it the way I thought.
I meant instead of `joinpath` `/` the implicit join path via division operator

Reply:
Ah ok, I see, yes I can add that.